src/Kmd.Logic.FileSecurity.Client/filesecurity.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
```

Comment:
Why not use latest openapi spec i.e. 3.1 here?

Reply:
To use openapi 3.0.1 we need to use the latest autorest version as the autorest version (2.0 or legacy version) which we are using will not support openapi 3.0.1. Using autorest latest version to generate code gave some issues and it is different the way code is generated using legacy version. Few e.g. 
1. It was creating a separate csproj file outside the current project folder.
2. It was throwing few build errors which is not the case when legacy version is used.
3. It was not generating an client interface. It was generating a class directly.
We need to analyze these issues before going for latest autorest and openapi version. Hence in order to proceed with current client creation we went for openapi v2.
We can try to find the right way to use latest versions and then update later. :)